### PR TITLE
resections and reorganizes the text.

### PR DESCRIPTION
**Pull request by hastinarimanzadeh**
- Resections and reorganizes the sections.
- Some material is excluded to be included in the `serial jobs` tutorial.
- Adds a little more code execution.
- The exercise session is left to be modified.

### CODE REVIEW

File: triton/tut/interactive.rst [hunk 1]
```diff
@@ -2,160 +2,190 @@
...
-properly, it is inefficient because you request more than you need. You
-should start here, but once you need more go to more advanced usage.
+computer nodes. Hundreds of people are using Triton simultaneously.
+Thus, resources (CPU time, memory, etc.) need to be shared among everyone.
 
-.. note::
+This resource sharing is done by a software called a job scheduler or
+workload manager. Triton's workload manager is **slurm**. 
+Triton users submit jobs which are then scheduled and allocated
+resources by the workload manager, slurm.
```
Comment (rkdarst) [inline triton/tut/interactive.rst:15]:
```suggestion
resources by the workload manager. 
```

File: triton/tut/interactive.rst [hunk 2]
```diff
@@ -2,160 +2,190 @@
...
-the process (see below).**
+.. warning::
 
-If you are off-campus, you might want to use https://vdi.aalto.fi as a
-virtual desktop to connect to Triton to run graphical programs.
-Otherwise, expect things to be very slow.
+  Just like with ``srun --pty bash``, remember to exit the shell.
+  Since there is a separate screen session running, just closing the terminal isn't enough. 
+  Exit all shells in the screen session on the node (C-d or ``exit``), or cancel
+  the process (see below).
```
Comment (rkdarst) [inline triton/tut/interactive.rst:137]: I think it actually does exit properly, I was just either wrong or or it was broken and I fixed it.  Want to test and if it does the right thing, remove this?
```suggestion
```

File: triton/tut/interactive.rst [hunk 3]
```diff
@@ -2,160 +2,190 @@
...
-important.  Partitions are groups of nodes reserved for different
-purposes.  For example, ``-p interactive`` tells us to us the
-interactive partition - which should always be available for quick
-tests.  ``-p debug`` is a short partition for debugging.  See a bit
-more in the :doc:`serial tutorial <serial>`.
+When your jobs enter the queue, you need to be able to get
+information on how much time, memory, etc. your jobs are using 
+in order to know what requirements to ask for. 
 
+The command ``slurm h`` gives you information such as the actual memory used by your recent jobs, total CPU time, etc.
```
Comment (rkdarst) [inline triton/tut/interactive.rst:152]:
```suggestion
The command ``slurm history`` gives you information such as the actual memory used by your recent jobs, total CPU time, etc.
```
Full command to make it easier to remember?  Which do you think is easier?

File: triton/tut/interactive.rst [hunk 4]
```diff
@@ -2,160 +2,190 @@
...
+You will learn more about these command later on. 
 
+As shown in a previous example, the command ``slurm q`` will tell you the currently running processes,
```
Comment (rkdarst) [inline triton/tut/interactive.rst:155]:
```suggestion
As shown in a previous example, the command ``slurm queue`` will tell you the currently running processes,
```

File: triton/tut/interactive.rst (hunk 3 above)
Comment (hastinarimanzadeh) [reply triton/tut/interactive.rst:152]: Ah yes. I could write the full command and then also introduce the short one.

File: triton/tut/interactive.rst (hunk 2 above)
Comment (hastinarimanzadeh) [reply triton/tut/interactive.rst:137]: The interactive partition is either drained or busy. checking periodically.

File: triton/tut/interactive.rst [hunk 5]
```diff
@@ -2,160 +2,185 @@
...
-So, let's say you need to do something a bit fancier: what if you want
-an actual shell to do things interactively? You just need the extra
-``--pty`` and run ``bash``.  The ``-p interactive`` says "give me a
-partition dedicated to interactive usage (more on this later).  Full
-example::
+What if you want an actual shell to do things interactively? 
+Put more precisely, you want access to a node in the cluster
+through an interactive bash shell. 
+For this, you just need srun's ``--pty`` option coupled with the shell
+you want
```
[the remark this answers is not in the document]
Comment (hastinarimanzadeh) [reply triton/tut/interactive.rst:91]: I kind of liked the transitioning from srun to --pty. But I can't say which one may be easier to start with.
Let's talk about it later then.

**Pull request merged**